<launch>
    <!-- <node pkg="rosserial_arduino" type="serial_node.py" name="rosserial_arduino" output="log" respawn="false">
        <param name="port" value="/dev/ttyACM0"/>
        <param name="baud" value="115200"/>
    </node> -->
    <node pkg="tf2_ros" type="static_transform_publisher" name="nav_frame" args="0 0 0 0 0 0 nav odom"/>
    <node pkg="tf2_ros" type="static_transform_publisher" name="map_frame" args="0 0 0 1.57 0 0 nav map"/>
    <node pkg="autonet_r1" type="nav.py" name="nav" output="screen" clear_params="true">
        <param name="config" value="$(find autonet_r1)/src/nav/nav_config.json"/>
        <param name="type" value="sim"/>
    </node>
    <node pkg="autonet_r1" type="navigate.py" name="navigate" output="screen" clear_params="true">
        <param name="config" value="$(find autonet_r1)/src/nav/navigate_config_sim.json"/>
        <param name="motors_config" value="$(find autonet_r1)/src/motors/config.json"/>
    </node>
    <node pkg="autonet_r1" type="forward.py" name="forward" output="screen" clear_params="true">
        <param name="config" value="$(find autonet_r1)/src/nav/navigate_config_sim.json"/>
        <param name="motors_config" value="$(find autonet_r1)/src/motors/config.json"/>
    </node>
    <node pkg="autonet_r1" type="get_path.py" name="get_path" output="screen" clear_params="true">
        <param name="map" value="$(find autonet_r1)/src/maps/map_1.json"/>
        <param name="map_coordinates" value="$(find autonet_r1)/src/maps/map_coordinates_1.json"/>
    </node>
</launch>